pkgs/development/coq-modules/bignums/default.nix

```diff
@@ -8,6 +8,7 @@ with lib; mkCoqDerivation {
   defaultVersion = if versions.isGe "8.5" coq.coq-version
     then "${coq.coq-version}.0" else null;
 
+  release."8.14.0".sha256 = "sha256:0jsgdvj0ddhkls32krprp34r64y1rb5mwxl34fgaxk2k4664yq06";
```

Comment:
Is the `sha:` prefix intended?

Reply:
Lately, I've stopped using the `nix-prefetch-url` tool to favor the `lib.fakeSha256` technique and copying the output of the expected hash. This is how it gets printed nowadays.

Reply:
Good to know. Thanks.